### PR TITLE
Feat/in 2450/whitelist volume metrics

### PR DESCRIPTION
**Pull request by thorfour**

### CODE REVIEW

File: pkg/decorate/compat/names.go [hunk 1]
```diff
@@ -36,8 +19,8 @@ func (n Names) Name() string {
 func (Names) Decorate(mfs []*dto.MetricFamily) {
 	for _, mf := range mfs {
 		n := strings.ToLower(mf.GetName())
-		if newName, ok := nameConversions[n]; ok {
-			mf.Name = &newName
+		if strings.HasPrefix(n, "node_") {
```
Comment (bsnyder788) [inline pkg/decorate/compat/names.go]: are we sure there weren't other metrics we didn't want name converted  before  that will be now?

Comment (bsnyder788) [reply pkg/decorate/compat/names.go]: e.g. I see "node_cpu_seconds_total" as a metric, that wasn't name converted before afaict

Comment (thorfour) [reply pkg/decorate/compat/names.go]: I believe it should have been all our `node_` metrics

Comment (thorfour) [reply pkg/decorate/compat/names.go]: nevermind there are other problems with doing a prefix replace

Comment (thorfour) [reply pkg/decorate/compat/names.go]: going back to the old map LUT

**Pull request merged**